virtual/httpd-cgi/httpd-cgi-0-r5.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Virtual for CGI-enabled webservers"
+
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x64-solaris"
```

Comment:
Cannot push directly to stable, please run `ekeyword ~all httpd-cgi-0-r5.ebuild` (from `app-portage/gentoolkit`)